_search-plugins/knn/performance-tuning.md

```diff
@@ -77,13 +132,15 @@ Take the following steps to improve search performance:
 
    The warmup API operation loads all native library indexes for all shards (primary and replica) for the specified indexes into the cache, so there's no penalty to load native library indexes during initial searches.
 
-   **Note**: This API operation only loads the segments of the indexes it ***sees*** into the cache. If a merge or refresh operation finishes after the API runs, or if you add new documents, you need to rerun the API to load those native library indexes into memory.
+This API operation only loads the segments of the indexes it ***sees*** into the cache. If a merge or refresh operation finishes after the API runs, or if you add new documents, you need to rerun the API to load those native library indexes into memory.
```

Comment:
The first sentence here is a bit awkward because the operation can't literally "see" anything, including index segments. Can we revise for clarity?

Reply:
How about "active" indexes?